app/components/exp-add-frame.js

```diff
@@ -0,0 +1,30 @@
+import ExpFrameBaseComponent from 'exp-player/components/exp-frame-base';
```

Comment:
I'm not sure if ember has best practices, but it would be great if somewhere in the component, we had a docstring describing what this component is intended to do.

(Mostly relevant as I try to keep straight all the things with "exp", "editor", and "frame" in their names, but also helpful for identifying components with potential reuse value as the project grows)